### PR TITLE
CSS added for #5 & #6

### PR DESCRIPTION
**Pull request by alvarotorrestx**
- Added CSS for Issues #5 & #6 
- added css aligning for chat.css
- added css alignment and style for input.css (button also - wasn't sure if wanted)
- added import for input.css on the input.jsx as it was missing
- updated index.js as it was crashing on local running for not finding 'app.js' <- not sure if only a local problem

### CODE REVIEW

**Comment by toth2000** [comment]
Thanks for the PR. I will check and merge it soon ( I have an exam today, so I will check later).  And please always make changes on code into a separate branch (other than main/master) while making PR in any repository as it is recommended by everyone.

**Comment by alvarotorrestx** [comment]
Definitely will! Thank you for that tip, will proceed to do further on further PRs!

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './app';
+import App from './App';
```
Comment (toth2000) [inline src/index.js:3]: this should be 
```
import App from './app';
```

**Review by toth2000 — CHANGES_REQUESTED** [review]
Any reason why you have deleted 22,382 lines of code from the package-lock.json file?
Other than that everything looks fine.

Please make the suggested changes on the branch that you have used on this pull request.
Let me know once done.

File: src/index.js (hunk 1 above)
Comment (alvarotorrestx) [reply src/index.js]: Thank you, updated!

**Comment by alvarotorrestx** [comment]
> Any reason why you have deleted 22,382 lines of code from the package-lock.json file?
> Other than that everything looks fine.
> 
> Please make the suggested changes on the branch that you have used on this pull request.
> Let me know once done.

I'm really not sure what happened on that deletion or where it came from, I see all the files but I do see where it marks it as deleted, I do apologize for that, I'll look further into it

As per the corrections after review, I have submitted that and thank you!

I'm still learning how to fully use git and github to the best ability so I do apologize for any delay

**Comment by toth2000** [comment]
@alvarotorrestx  No worries! You are doing great. And Thank you for your contribution.  I will merge it soon.

**Comment by toth2000** [comment]
Merging this branch. Thank you for your contributions.!!

**Comment by alvarotorrestx** [comment]
Definitely welcome and thank you for letting me contribute! I'll continue to follow to see if I'm able to continue helping! 😁

**Pull request merged**